apps/address-service/domains/address/schema/Address.test.js

```diff
@@ -178,4 +184,59 @@ describe('Address', () => {
             })
         })
     })
+
+    describe('Use cases', async () => {
+        test('The house keeps the firstly created address', async () => {
+            const source1 = faker.random.alphaNumeric(42)
+            const source2 = faker.random.alphaNumeric(42)
+            const address1 = faker.random.alphaNumeric(42)
+            const address2 = faker.random.alphaNumeric(42)
+
+            const fakeSearchResult = {
+                value: '',
+                unrestricted_value: '',
+                data: {
+                    country: faker.random.alphaNumeric(12),
```

Comment:
faker has random for country, street, address, etc

Reply:
Yes, fixed.